nilearn/_utils/docs.py

```diff
@@ -723,7 +703,7 @@ def custom_function(vertices):
 
         - `ridge_classifier`: `{rc} <%(ridge_classifier)s>`_.
             .. code-block:: python
-
+g
```

Comment:
What is this `g` doing there?

Reply:
no idea: my only excuse is that I am not using my regular keyboard so I am having a lot of misfires...